libs/indicom.h

```diff
@@ -370,11 +370,11 @@ double m2au(double m);
 double calc_delta_magnitude(double mag_ratio, double *spectrum, double *ref_spectrum, int spectrum_size);
 
 /**
- * @brief calc_photon_flux Returns the photon flux of the object with the given magnitude observed at a determined wavelenght using a passband filter over an incident surface
+ * @brief calc_photon_flux Returns the photon flux of the object with the given magnitude observed at a determined wavelenght using a passband filter through a steradian expressed cone
  * @param rel_magnitude Relative magnitude of the object observed
  * @param filter_bandwidth Filter bandwidth in meters
  * @param wavelength Wavelength in meters
- * @param incident_surface The incident surface in square meters
+ * @param steradian The light cone in steradians
  * @return the photon flux in Lumen
  */
 double calc_photon_flux(double rel_magnitude, double filter_bandwidth, double wavelength, double incident_surface);
```

Comment:
last parameter needs to be renamed?

Reply:
please wait some minutes I'm working on this right now. sorry.

Reply:
Can you re-open the interferometer PR in 3rdparty?

Reply:
Allright, this is mergeable

Reply:
This one can be merged, thanksfor your patience.